examples/graphs/hopcroft_karp_aizu.rs

```diff
@@ -0,0 +1,47 @@
+// verification-helper: PROBLEM https://onlinejudge.u-aizu.ac.jp/problems/GRL_7_A
+
+use proconio::input;
+use programming_team_code_rust::graphs::hopcroft_karp::get_max_matching;
```

Comment:
hopcroft_karp vs max_matching, why not just use 1? if you like this naming that's ok though